peripherals/tim/tim21.yaml

```diff
@@ -2,10 +2,6 @@
 
 "TIM2?":
   CR1:
-    CKD:
-      Div1: [0, "t_DTS = t_CK_INT"]
-      Div2: [1, "t_DTS = 2 × t_CK_INT"]
-      Div4: [2, "t_DTS = 4 × t_CK_INT"]
```

Comment:
As mentioned in the commit comment, this is already present in `tim_basic.yaml`. `tim21.yaml` is _only_ used in `tim_l0` currently so I don't think this should cause any issues.